app/code/Magento/Cms/Test/Mftf/Test/AdminCreateDuplicatedCmsPageTest.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateDuplicatedCmsPageTest">
+        <annotations>
+            <features value="Cms"/>
+            <stories value="CMS Page Duplication and Reset Removal MAGETWO-87096"/>
+            <title value="Admin should be able to duplicate a CMS Page"/>
+            <description value="Admin should be able to duplicate a CMS Page"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MAGETWO-89184"/>
+            <group value="Cms"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginActionGroup" stepKey="loginGetFromGeneralFile"/>
+            <actionGroup ref="DisabledWYSIWYGActionGroup" stepKey="disableWYSIWYG"/>
```

Comment:
@rogyar I don't remember, but it's no longer necessary to add this. How do you think?